python/tk_desktop/extensions/darwin_python2/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2020 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
```

Comment:
There are the missing `__init__.py` that prevented the `osutils.so` file from being loaded in the `darwin_python2` and `darwin_python3` folders.